test/Bitcoin/AddressSpec.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE GADTs #-}
```

Comment:
What's this for?

Reply:
Used in lieu of -XExistentialQuantification for the below `forall`s that appear in the various boxes.